draft-ietf-quic-transport.md

```diff
@@ -1453,6 +1453,10 @@ return.  A PING frame containing 12 randomly generated {{?RFC4086}} octets is
 sufficient to ensure that it is easier to receive the packet than it is to guess
 the value correctly.
 
+Note:
+
+: Retransmissions of the PING frame MUST also use the same remote address.
+
 If validation of the new remote address fails, after allowing enough time for
```

Comment:
Ahh, that isn't necessarily your fault...

I think that this is OK, given the qualification about abandoning validation attempts.